packages/client/core/src/lib/CryptidTransaction.ts

```diff
@@ -119,43 +135,52 @@ export class CryptidTransaction {
       DID_SOL_PROGRAM,
       authority,
     ];
-    const remainingAccounts = transactionAccount.accounts;
-    const allAccounts = [...namedAccounts, ...remainingAccounts];
-    const accountMetas = transactionAccountMetasToAccountMetas(
+    const allAccounts = transactionAccount.accounts;
+    const accountMetasFromInstructions = transactionAccountMetasToAccountMetas(
       instructions.flatMap((i) => [
         ...(i.accounts as TransactionAccountMeta[]),
         { key: i.programId, meta: 0 },
       ]),
       allAccounts,
       cryptidAccount
     );
-    const filteredAccountMetas = accountMetas.filter(
-      (a) =>
-        !namedAccounts
-          .map((account) => account.toBase58())
-          .includes(a.pubkey.toBase58())
-    );
+    const filteredAccountMetasFromInstructionsExcludingNamedAccounts =
```

Comment:
I'm really trying to decide if that renaming makes it clearer or not :).

Reply:
`ExludingNamedAccounts == RemainingAccounts`

Reply:
renamed again